Overhaul setup guides (#256)
SkyTemple/c-of-time: install_linux.md

# Linux setup guide

> You can install c-of-time on Windows using the following methods. Instructions vary based on your Linux distribution.

Reviewer: ```suggestion
You can install c-of-time on Linux using the following methods. Instructions vary based on your Linux distribution.
```

2. Run the following command to install the required tools: `sudo apt install build-essential cmake python3-pip gcc-arm-none-eabi binutils-arm-none-eabi`. You will need to enter your password during the installation.
3. Install Python dependencies: `pip3 install pyyaml ndspy`
4. Compile `armips`:
    1. Run `git clone --recursive https://github.com/Kingcom/armips.git` to download the source code.
    2. Run `cd armips` to enter the directory.
    3. Run this command to compile the project: `mkdir build && cd build && cmake -DCMAKE_BUILD_TYPE=Release .. && cmake --build .`
    4. Run `sudo cp armips /usr/local/bin` to install `armips`.
    5. Run `cd ../..` to return to the previous directory.
5. Navigate to the directory where you want to install c-of-time. You can use `cd` to change the directory and `ls` to list the contents of the current directory. For example, if you want to install c-of-time to `/home/YourName/Documents/c-of-time`, run `cd /home/YourName/Documents`.
    - **Note:** You can also use the file manager to navigate to the directory where you want to install c-of-time. Right-click the name of the directory and select "Open in Terminal".

> 6. Download this repository by running `git clone --recursive https://github.com/SkyTemple/c-of-time.git` in the MSYS2 terminal. c-of-time will be downloaded in a folder called `c-of-time` inside the current directory.

Reviewer: ```suggestion
6. Download this repository by running `git clone --recursive https://github.com/SkyTemple/c-of-time.git` in the terminal. c-of-time will be downloaded in a folder called `c-of-time` inside the current directory.
```

7. Enter the `c-of-time` directory with `cd c-of-time`.

> 8. Copy the ROM you have prepared into the `c-of-time` directory and rename it to `rom.nds`. You can open Windows Explorer in the current directory by running `explorer .` in the MSYS2 terminal.

Reviewer 2: ah yes my wine port of explorer.exe :P

Reviewer 2: Under Linux you can use `xdg-open .`.

Reviewer: So probably

```suggestion
8. Copy the ROM you have prepared into the `c-of-time` directory and rename it to `rom.nds`. You can open the file manager in the current directory by running `xdg-open .` in the terminal.
```

Reviewer 2: I always forget suggesting commits is a thing.

Author: That's some very sloppy copy-pasting from my side, thanks.
